### PR TITLE
Updated contactform file & added database to files

### PR DESCRIPTION
**Pull request by yellow-tulips**
I updated the email for the contactform.php file.
I added PHP code to all of the original HTML files for database purposes.

### CODE REVIEW

**Review by joerossi215 — APPROVED** [review]
Looks good to go!!

File: about.php [hunk 1]
```diff
@@ -0,0 +1,184 @@
+<?php 
+   session_start();
+?>
+<!DOCTYPE html>
+<html>
+
+<head>
+    <meta charset="utf-8">
+    <title>About Us</title>
```
Comment (Taraque-A) [inline about.php:9]: Nice!

File: about.php [hunk 2]
```diff
@@ -0,0 +1,184 @@
...
+</head>
+<style>
+    @import url(styleabout.css);
+</style>
+<script src="https://cdnjs.cloudflare.com/ajax/libs/animejs/2.0.2/anime.min.js"></script>
+
+<body>
+    <div id="background-wrap">
+        <div class="x1">
+            <div class="cloud"></div>
```
Comment (Taraque-A) [inline about.php:19]: Class looks nice!

File: about.php [hunk 3]
```diff
@@ -0,0 +1,184 @@
...
+    			        <?php echo '<li><a href="logout.php">Logout</a></li>' ?>
+    		        <?php } ?>
+
+            </ul>
+        </nav>
+    </header>
+    
+    <div class = "aboutus">
+    <div class = "mission">
+    <h2>Our Mission</h2>
```
Comment (Taraque-A) [inline about.php:61]: Really nice!

File: about.php [hunk 4]
```diff
@@ -0,0 +1,184 @@
...
+        </p>
+        <br>
+    </div>
+
+
+
+<br><br>
+
+<div class="who">
+<h2>Who We Are</h2>
```
Comment (Taraque-A) [inline about.php:78]: This is awesome!

File: about.php [hunk 5]
```diff
@@ -0,0 +1,184 @@
...
+    This is my senior year and I have one more semester left at Wayne State University. The things I like about being a software engineer must always learn new things. 
+    Every other day, a new advancement in this subject occurs due to the ongoing evolution of technology. As a result, one of the benefits of becoming a software engineer 
+    is having the opportunity to pick up new skills at any moment. My hobbies are spending time with family, friends, travel, football, basketball, soccer, and hockey.</p>
+
+</div>
+</div>
+
+<br><br>
+
+<h2>Contact Us</h2>
```
Comment (Taraque-A) [inline about.php:107]: Looks great!

File: about.php [hunk 6]
```diff
@@ -0,0 +1,184 @@
...
+<p class="contactgreeting">We are open to answering questions anyone may have regarding our mission, features, or our overall website.</p>
```
Comment (Taraque-A) [inline about.php:108]: This is look pretty good!

File: about.php [hunk 7]
```diff
@@ -0,0 +1,184 @@
...
+            <label for="number">Phone Number</label>
+        </div>
+      <div class="col-75">
+        <input type="text" name="usernumber" id="form-num" placeholder="Enter your phone number">
+      </div>
+      </div>
+      
+      <div class="row">
+        <div class="col-25">
+          <label for="subject">Feedback or Question</label>
```
Comment (Taraque-A) [inline about.php:162]: Really cool!

File: discussion.php [hunk 8]
```diff
@@ -0,0 +1,142 @@
+<?php 
+   session_start();
```
Comment (Taraque-A) [inline discussion.php:2]: This is really a great idea!

File: discussion.php [hunk 9]
```diff
@@ -0,0 +1,142 @@
...
+<body>
+    <h1 style="text-align: center;">Discussion Forum</h1>
+
+<form action="discussion.php" method="POST">
+<?php if (isset($message)) { ?>
+            <p><?php echo $message; ?></p>
+            <?php } ?>
+
+
+<textarea name="comment" id="comment" rows="10" cols="150" maxlength="1000" style="border:8px outset rgb(226, 146, 159);" placeholder="Talk about tips, concerns, or anything else you would like to share"></textarea>
```
Comment (Taraque-A) [inline discussion.php:101]: Great!

File: logout.php [hunk 10]
```diff
@@ -0,0 +1,5 @@
+<?php
```
Comment (Taraque-A) [inline logout.php:1]: Great job!

**Review by Taraque-A — APPROVED** [review]

**Review by farzanaisrat — APPROVED** [review]
looks good!

**Review by tasnhubaislam — APPROVED** [review]
Everything looks good

**Pull request merged**